app/src/main/assets/applozic-settings.json

```diff
@@ -150,5 +150,9 @@
     "sendMessageOnSpeechEnd": true,
     "languageCode": "en-US"
   },
-  "hidePostCTA": false
+  "hidePostCTA": {
```

Comment:
What will happen to existing customers using old structure in json file and upgrades SDK without changing json file

Reply:
Nothing will happen. This is not live yet.